static/js/tools/stat_var/page.tsx

```diff
@@ -58,6 +69,22 @@ class Page extends Component<unknown, PageStateType> {
       this.fetchSummary();
     };
     this.fetchSummary();
+    const sourcePromise = await axios.get(
```

Comment:
for both this promise and the sourceNamesPromise, we should add some error handling

Reply:
done